reports/report_emailer.py

```diff
@@ -114,21 +99,25 @@ def get_delay_and_cancellation_percentages(conn: psycopg2.extensions.connection,
         num_of_delays[0])/(int(num_of_arrivals[0]+int(num_of_cancellations[0])))*100, 2)
 
     cancellation_percentage = round(int(
-        num_of_cancellations[0])/(int(num_of_arrivals[0]+int(num_of_cancellations[0])))*100, 2)
+        num_of_cancellations[0])/(int(num_of_arrivals[0]+int(num_of_cancellations[0])))*100,
+        2)
 
-    return {"delay_percentage": delay_percentage, "cancellation_percentage": cancellation_percentage}
+    return {"delay_percentage": delay_percentage,
+            "cancellation_percentage": cancellation_percentage}
 
 
 def get_average_delay_time(conn: psycopg2.extensions.connection, station_crs: str) -> float:
     """This function returns the average delay time for a station given the station's
     crs code."""
 
-    avg_delay_query = f"""SELECT 
+    avg_delay_query = f"""SELECT
                         ROUND(AVG((EXTRACT(EPOCH FROM (arrivals.actual_arrival - arrivals.scheduled_arrival)) / 60)), 2) AS average_delay_minutes 
                         FROM arrivals
                         INNER JOIN stations
                         ON stations.station_id=arrivals.station_id
                         WHERE stations.crs_code='{station_crs}'
+                        AND 
+                        arrivals.actual_arrival > arrivals.scheduled_arrival
```

Comment:
If you prefer, you can store all of your SQL queries in separate Python file (such as sql_queries.py) as variables can indeed make your code more organised and easier to read